recipe/meta.yaml

```diff
@@ -39,17 +42,17 @@ test:
     - pip
 
 about:
-  home: https://hvplot.holoviz.org/
+  home: https://hvplot.holoviz.org
   license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE
   summary: A high-level plotting API for the PyData ecosystem built on HoloViews
   dev_url: https://github.com/holoviz/hvplot
   doc_url: https://nbsite.holoviz.org/
```

Comment:
Here is the up-to-date docs https://hvplot.holoviz.org/user_guide/index.html

Reply:
Yes, that `doc_url` is incorrect. I would just point it to  https://hvplot.holoviz.org as well.